cirq/contrib/json.py

```diff
@@ -0,0 +1,15 @@
+"""Contains functions for adding JSON serialization and de-serialization for
+classes in Contrib.
+
+"""
+
+
+def contrib_class_resolver(cirq_type: str):
+    """Extend cirq's JSON API with resolvers for cirq contrib classes."""
+    from cirq.contrib.quantum_volume import QuantumVolumeResult
+    classes = [
+        QuantumVolumeResult,
+    ]
+    print(QuantumVolumeResult.__name__)
```

Comment:
Leftover debug line

Reply:
D'oh I fixed that and forgot to commit it :P Done.